renderer/types/Charts.ts

```diff
@@ -0,0 +1,26 @@
+export type DataRelatedChart = {
```

Comment:
these are props for a specific component, so it should be called DataRelatedChartProps, you could make it into an interface like this

```ts
interface DataSize {
  size: number;
  unit: "gb" | "tb" | "bytes"; // better be more specific than just 'string'
}
```

export interface DataRelatedChart {
  data_package: DataSize;
  storage_available: DataSize;
  total_size: DataSize;
}
``` 